pyanaconda/iutil.py

```diff
@@ -48,7 +48,7 @@
 from gi.repository import GLib
 
 from pyanaconda.flags import flags
-from pyanaconda.constants import DRACUT_SHUTDOWN_EJECT, TRANSLATIONS_UPDATE_DIR, UNSUPPORTED_HW, IPMI_ABORTED
+from pyanaconda.constants import DRACUT_SHUTDOWN_EJECT, TRANSLATIONS_UPDATE_DIR, UNSUPPORTED_HW, IPMI_ABORTED, X_TIMEOUT
```

Comment:
One last small thing, could you please wrap this line to 99 characters before merge. Thanks.

Reply:
Fixed.